aptos-move/framework/aptos-token/doc/token.md

```diff
@@ -564,7 +564,7 @@ This config specifies which fields in the Collection are mutable
 <code>maximum: bool</code>
 </dt>
 <dd>
- control if collection maxium is mutable
+ control if collection maximum is mutable
```

Comment:
these must be made directly in the corresponding `.move` file and then code gen re-run easiest by running `cargo build` in the repo root.